django/template/loaders/base.py

```diff
@@ -44,3 +93,10 @@ def reset(self):
         templates or cached loader modules).
         """
         pass
+
+    @property
+    def supports_recursion(self):
+        """
+        Used during deprecation period of non-recursive loaders.
```

Comment:
Non-recursive loaders are ones that only define ``load_template_sources``. The deprecation is noted here:

https://github.com/django/django/pull/3475/files#diff-c0f4f3a0c26f12ebc2e41904af710f4dR361

Do you think this just needs a clearer doc string? Or is there something else I should add?